api/v1/ytsaurus_types.go

```diff
@@ -779,15 +782,18 @@ const (
 	UpdateSelectorEverything UpdateSelector = "Everything"
 )
 
-type UpdateFlow string
+type ComponentUpdateSelector struct {
+	//+optional
+	ComponentType consts.ComponentType `json:"componentType,omitempty"`
+	//+optional
+	ComponentGroup consts.ComponentGroup `json:"componentGroup,omitempty"`
+	//+optional
+	ComponentName string `json:"componentName,omitempty"`
```

Comment:
I have no idea what to put in the Name actually. 
I suggest we add examples here for unnamed component like master, for unnamed datanode and for named datanode.

Also AFAIK we have `Name` which already include type and for sure users will try to put the value which they set in the spec for the respective component. 
Let's think now if we really want "full" names here. If so I guess we should name the field like FullName or smth.